modules/social_features/social_user/src/Plugin/QueueWorker/UserMailQueueProcessor.php

```diff
@@ -114,7 +114,9 @@ public function processItem($data) {
           /** @var \Drupal\user\UserInterface $user */
           foreach ($users as $user) {
             // Attempt sending mail.
-            $this->sendMail($user->getEmail(), $user->language()->getId(), $queue_storage);
+            if ($user->isAuthenticated() || $user->getEmail()) {
```

Comment:
```suggestion
            if ($user->getEmail()) {
```

Shouldn't this just be getting the email? What if someone is authenticated and doesn't have an email address? It will still crash right?

Reply:
That's going to be highly unlikely case 😅 , but yes, the suggestion will take care of it :D